packages/RatPack.package/RPModel.class/class/updateObject.withJson..st

```diff
@@ -0,0 +1,11 @@
+object conversion
+updateObject: aRPModelClassObject withJson: aJsonString
+
+	| newObject aJsonObject|
+	
+	aJsonObject := self createJsonObject: aJsonString. 
```

Comment:
Since we also want to allow partial json objects here, we'd have to change something here. See comment above, `createJsonObject` currently forces the json to contain all attributes.

Reply:
No the test failure was a trivial thing because of the merge with @jakobmache branch with the RouteNaming #84 
Was a one line fix in our self written test. (The route in the beginning was false -> notFound -> WrongArgumentError #82 here not fixed -> error).

Reply:
Now I'm only iterating over the keys of the updateJsonObject (which wass already validated, so can't contain bad keys.

Reply:
Well the validation of the input still has to happen, and it still has to be different from the one we do for e.g. POST requests. Since `createJsonObject` calls `validateJson` and that forces us to give all non-default values, which we don't want, we will have to do something different here.

Actually, if this does not make any test fail, we should add a test that fails with this code.

Reply:
Yes there are now 2 validation methods, forgot to mention that. The described is the new for put.